src/mocks/timeseriesList.ts

```diff
@@ -0,0 +1,343 @@
+/* tslint:disable */
```

Comment:
We should avoid disabling an entire file

Reply:
Changed to disable specific rule